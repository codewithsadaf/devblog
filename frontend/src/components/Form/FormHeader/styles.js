//imports
import styled, { css } from "styled-components";
//components styles
import { TextComponent } from "../../Layout/TextComponent";

export const Container = styled.div`
    ${({ theme }) => css`
        margin-bottom: 1em;
    `}
`;

export const Text = styled(TextComponent)`
    margin-top: 0.3em;
`;
