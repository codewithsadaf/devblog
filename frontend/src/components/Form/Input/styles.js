//imports
import styled, { css } from "styled-components";

export const FormControl = styled.div`
    display: flex;
    flex-direction: column;
    margin-bottom: 1em;
`;

export const LabelComponent = styled.label`
    display: block;
    margin-bottom: 0.3em;
    font-weight: 700;
`;

export const InputComponent = styled.input`
    ${({ theme }) => css`
        padding: 7px 0;
        outline: none;
        border: none;
        border-bottom: 1px solid ${theme.colors.gray};
        background-color: transparent;
        font-size: ${theme.font.sizes.default};
        transition: ${theme.transition.tran_05};

        &:focus {
            border-bottom: 1px solid ${theme.colors.primaryColor};
            transition: ${theme.transition.tran_03};
        }
    `}
`;
