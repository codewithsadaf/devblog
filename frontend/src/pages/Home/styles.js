//imports
import styled, { css } from "styled-components";

export const HomeContainer = styled.div`
    ${({ theme }) => css`
        width: 600px;
        margin: 3em auto 0;

        @media ${theme.media.lteMedium} {
            max-width: 100%;
        }
    `}
`;

export const MessageWrapper = styled.div`
    margin-top: 5em;
    text-align: center;
    display: flex;
    align-items: center;
    justify-content: center;
`;
