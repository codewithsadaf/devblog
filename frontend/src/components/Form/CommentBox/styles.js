//imports
import styled, { css } from "styled-components";
//components styles
import { InputComponent } from "../Input/styles";
import { ButtonContainer as Button } from "../../Layout/Button/styles";

export const Container = styled.div`
    display: flex;
    align-items: center;
    justify-content: space-between;
    width: 100%;

    & ${Button} {
        padding: 12px 16px;
    }
`;

export const CommentInput = styled(InputComponent)`
    ${({ theme }) => css`
        border: 1px solid ${theme.colors.gray};
        transition: ${theme.transition.tran_03};
        padding: 12px 7px;
        width: 90%;
        border-radius: 5px;
        font-size: 1em;
        margin-right: 7px;

        &:focus {
            border: 1px solid ${theme.colors.primaryColor};
            transition: ${theme.transition.tran_03};
        }
    `}
`;
