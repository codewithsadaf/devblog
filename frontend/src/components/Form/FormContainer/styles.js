//imports
import styled, { css } from "styled-components";
//components styles
import { ButtonContainer as Button } from "../../../components/Layout/Button/styles";

export const Container = styled.div`
    width: 100%;
    height: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
`;

export const Form = styled.form`
    ${({ theme }) => css`
        width: 450px;
        margin-top: 3em;

        & > ${Button} {
            margin-top: 1em;
        }

        @media ${theme.media.lteMedium} {
            width: 100%;
            min-width: 0;
            max-width: 100%;
        }
    `}
`;
