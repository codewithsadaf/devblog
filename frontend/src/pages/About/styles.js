//imports
import styled, { css } from "styled-components";
//components styles
import { Text as TextComponent } from "../../components/Layout/TextComponent/styles";
import { Container as Heading } from "../../components/Layout/Heading/styles";

export const AboutContainer = styled.div`
    ${({ theme }) => css`
        max-width: 60%;
        margin: 3em auto 0;
        text-align: center;

        & > ${TextComponent} {
            margin-top: 0.5em;
        }

        @media ${theme.media.lteMedium} {
            max-width: 100%;

            & > ${Heading} {
                font-size: ${theme.font.sizes.xxlarge};
                line-height: ${theme.font.line_height.h3};
            }
        }
    `}
`;
