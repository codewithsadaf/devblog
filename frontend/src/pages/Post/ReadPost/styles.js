//imports
import styled, { css } from "styled-components";
import { Link } from "react-router-dom";
//components styles
import { ImageElm } from "../../../components/Layout/ImageBox/styles";
import { Text as TextComponent } from "../../../components/Layout/TextComponent/styles";

//PAGE CONTAINER
export const Container = styled.div`
    margin: 3em auto 0;
`;

//RETURN TO HOME TEXT
export const BackWrapper = styled.div`
    width: 100%;
    display: flex;
    align-items: center;
    justify-content: flex-start;
    margin-bottom: 1.2em;
    align-items: center;
`;

export const BackLink = styled(Link)`
    ${({ theme }) => css`
        color: ${theme.colors.primaryColor};
        transition: ${theme.transition.tran_03};
        display: flex;
        align-items: center;

        &:hover {
            transition: ${theme.transition.tran_03};
            opacity: 0.7;
        }
    `}
`;

//POST HEADER
export const PostHeader = styled.div`
    ${({ theme }) => css`
        width: 80%;
        margin: 0 auto;
        text-align: center;

        ${ImageElm} {
            height: 50%;
            width: 50%;
            margin: 2em auto;

            @media ${theme.media.lteMedium} {
                width: 100%;
                height: 100%;
            }
        }

        ${TextComponent} {
            margin: 1em 0;
        }

        @media ${theme.media.lteMedium} {
            width: 100%;
        }
    `}
`;

//POST CONTENT
export const ContentWrapper = styled.div`
    ${({ theme }) => css`
        width: 60%;
        margin: 0 auto;

        @media ${theme.media.lteMedium} {
            width: 100%;
        }
    `}
`;

export const DisplayContent = styled.div`
    ${({ theme }) => css`
        text-align: left;

        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin-bottom: 0.5em;
            font-weight: bold;
        }

        p {
            font-size: 18px;
            margin-bottom: 1em;
        }

        a {
            font-weight: bold;
            color: ${theme.colors.primaryColor};
            text-decoration: underline;
        }

        ol,
        ul {
            margin-left: 2em;

            li {
                margin-bottom: 1em;
            }
        }

        ul {
            list-style-type: disc;
        }

        ol {
            list-style-type: decimal;
        }

        img {
            width: 100%;
            display: block;
            margin: 2em 0;
        }
    `}
`;

//POST TAGS
export const TagsWrapper = styled.div`
    display: flex;
    justify-content: center;
    align-items: center;
    width: 100%;
    margin: 3em 0;
    flex-wrap: wrap;
`;

export const TagText = styled.span`
    display: block;
    padding: 0 0.5em;
    font-weight: 500;

    &::before {
        content: "#";
        font-weight: bold;
    }
`;

export const TagsInfo = styled.span`
    display: block;
    margin-right: 0.5em;
    font-weight: bold;
`;

//POST LINE
export const Line = styled.div`
    ${({ theme }) => css`
        height: 2px;
        width: 100%;
        background-color: ${theme.colors.footerBackground};
        margin: 1em 0;
    `}
`;

//POST COMMENTS
export const CommentsNumberText = styled.h5`
    ${({ theme }) => css`
        font-size: ${theme.font.sizes.large};
        line-height: ${theme.font.line_height.h5};
    `}
`;

export const CommentsWrapper = styled.div`
    width: 100%;
    display: flex;
    flex-direction: column;
`;

export const SignUpWrapper = styled.div`
    margin: 2em 0;
    display: flex;
    align-items: center;
    justify-content: space-between;
    flex-direction: column;
    width: 100%;
    text-align: center;

    button {
        margin-top: 1em;
    }
`;

export const SignUpText = styled.h4`
    ${({ theme }) => css`
        font-size: ${theme.font.sizes.xlarge};
        line-height: ${theme.font.line_height.h4};
    `}
`;
