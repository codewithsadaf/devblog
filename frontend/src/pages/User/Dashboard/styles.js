//imports
import styled, { css } from "styled-components";
//components styles
import { ImageElm } from "../../../components/Layout/ImageBox/styles";
import { Container as Heading } from "../../../components/Layout/Heading/styles";
import { ButtonContainer as Button } from "../../../components/Layout/Button/styles";
import { InputComponent } from "../../../components/Form/Input/styles";
//react icons
import { FaCog } from "react-icons/fa";
import { Link } from "react-router-dom";

export const Container = styled.div`
    ${({ theme }) => css`
        width: 80%;
        margin: 3em auto 0;

        @media ${theme.media.lteMedium} {
            width: 100%;
        }
    `}
`;

export const TitleWrapper = styled.div`
    display: flex;
    align-items: center;
    justify-content: center;
    width: 100%;
    margin-bottom: 1.2em;
`;

export const ProfileWrapper = styled.div`
    ${({ theme }) => css`
        display: flex;
        align-items: center;
        justify-content: space-between;

        @media ${theme.media.lteMedium} {
            flex-direction: column;
        }
    `}
`;

export const ProfileImageWrapper = styled.div`
    ${({ theme }) => css`
        flex-basis: 40%;

        ${ImageElm} {
            width: 300px;
            height: 300px;
            object-fit: fill;
            border-radius: 50%;

            @media ${theme.media.lteMedium} {
                width: 150px;
                height: 150px;
            }
        }

        @media ${theme.media.lteMedium} {
            flex-basis: 100%;
        }
    `}
`;

export const ProfileInformationWrapper = styled.div`
    ${({ theme }) => css`
        flex-basis: 58%;
        min-height: 250px;
        text-align: left;

        ${Heading}:nth-of-type(2) {
            margin: 0.3em 0;
        }

        @media ${theme.media.lteMedium} {
            flex-basis: 100%;
            width: 100%;
            margin-left: 0;
        }
    `}
`;

export const RowStart = styled.div`
    display: flex;
    align-items: center;
    justify-content: flex-start;
    flex-wrap: wrap;
`;

export const CogLink = styled(Link)`
    ${({ theme }) => css`
        font-size: 1.2em;
        margin-left: auto;
        cursor: pointer;
        color: ${theme.colors.primaryColor};
        opacity: 1;
        transition: ${theme.transition.tran_05};

        &:hover {
            opacity: 0.6;
            transition: ${theme.transition.tran_05};
        }
    `}
`;

export const CogIcon = styled(FaCog)`
    ${({ theme }) => css``}
`;

export const PostsCommentsInfoWrapper = styled.div`
    ${({ theme }) => css`
        display: flex;
        width: 100%;
        border: 1px solid ${theme.colors.gray};
        border-radius: 7px;
        padding: 20px;
        margin-top: 1em;
    `}
`;

export const PostsInfoWrapper = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    margin-right: 2em;
`;

export const CommentsInfoWrapper = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
`;

export const ViewOptionWrapper = styled.div`
    ${({ theme }) => css`
        margin: 2em 0;
        display: flex;
        border-bottom: 2px solid ${theme.colors.primaryColor};
    `}
`;

export const ViewOption = styled.span`
    ${({ theme }) => css`
        background-color: transparent;
        color: ${theme.colors.primaryColor};
        padding: 7px;
        font-size: ${theme.font.sizes.medium};
        border-radius: 7px 7px 0 0;
        font-weight: 500;
        cursor: pointer;
        transition: ${theme.transition.tran_03};

        &.active {
            background-color: ${theme.colors.primaryColor};
            color: ${theme.colors.white};
            transition: ${theme.transition.tran_03};
            pointer-events: none;
        }
    `}
`;

export const DashboardHeader = styled.div`
    ${({ theme }) => css`
        display: flex;
        justify-content: space-between;
        align-items: center;
        width: 100%;
        border-bottom: 2px solid ${theme.colors.gray};
        font-weight: 700;
        padding: 10px;
    `}
`;

export const DashboardText = styled.span`
    ${({ theme }) => css`
        font-size: ${theme.font.sizes.default};
        line-height: ${theme.font.line_height.default};
    `}
`;

export const DashboardRow = styled.div`
    ${({ theme }) => css`
        display: flex;
        justify-content: space-between;
        align-items: center;
        padding: 10px;
        width: 100%;
        border-bottom: 2px solid ${theme.colors.lightGray};

        & ${DashboardText} {
            display: block;
            width: 60%;
            word-wrap: break-word;

            @media ${theme.media.lteMedium} {
                width: 100%;
            }
        }

        @media ${theme.media.lteMedium} {
            flex-direction: column;
            justify-content: flex-start;
            align-items: flex-start;
        }
    `}
`;

export const DashboardActionsWrapper = styled.div`
    ${({ theme }) => css`
        margin-left: 2em;
        display: flex;
        justify-content: space-between;
        align-items: center;

        & ${Button} {
            margin-left: 0.5em;

            @media ${theme.media.lteMedium} {
                margin-left: 0;
                margin-right: 0.5em;
            }
        }

        @media ${theme.media.lteMedium} {
            margin-left: 0;
            margin: 1em 0;
        }
    `}
`;

export const CenterTextWrapper = styled.div`
    display: flex;
    align-items: center;
    justify-content: center;
    width: 100%;
    text-align: center;
    padding: 2em 0;
`;

export const EditInput = styled(InputComponent)`
    border: none;
    width: 100%;

    &:focus {
        border: none;
    }
`;
