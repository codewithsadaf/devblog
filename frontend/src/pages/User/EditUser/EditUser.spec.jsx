//imports
import { fireEvent, screen, waitFor } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import { act } from "react-dom/test-utils";
//styles
import { renderTheme } from "../../../styles/render-theme";
//contexts
import { Context as UserContext } from "../../../context/UserContext";
//components
import { EditUser } from "./";

//scrollTo mock
const spyScrollTo = jest.fn();
Object.defineProperty(global.window, "scrollTo", { value: spyScrollTo });

//localStorage mock
Object.defineProperty(window, "localStorage", {
    value: {
        getItem: jest
            .fn()
            .mockReturnValue(JSON.stringify({ token: "MockToken" })),
    },
});

describe("<UserEdit />", () => {
    let mockCheckUser;

    beforeEach(() => {
        mockCheckUser = jest.fn();

        mockCheckUser.mockReturnValue(
            Promise.resolve({
                currentUser: {
                    name: "john doe",
                    image: "image.png",
                    email: "john@test.com",
                },
            })
        );
    });

    describe("Rendering", () => {
        it("should render UserEdit", async () => {
            renderTheme(
                <BrowserRouter>
                    <UserContext.Provider
                        value={{
                            authenticated: true,
                            checkUser: mockCheckUser,
                        }}
                    >
                        <EditUser />
                    </UserContext.Provider>
                </BrowserRouter>
            );

            await waitFor(() => {
                expect(mockCheckUser).toHaveBeenCalledTimes(1);
            });

            const formHeading = screen.getByRole("heading", {
                name: "Editar Dados",
            });
            const formSubHeading = screen.getByText(
                "Atualize suas informações pessoais"
            );
            const imageInput = document.querySelector('input[name="image"]');
            const nameInput = document.querySelector('input[name="name"]');
            const emailInput = document.querySelector('input[name="email"]');
            const passwordInput = document.querySelector(
                'input[name="password"]'
            );
            const confirmPasswordInput = document.querySelector(
                'input[name="confirmpassword"]'
            );
            const EditUserButton = screen.getByRole("button", {
                name: "Atualizar",
            });

            expect(formHeading).toBeInTheDocument();
            expect(formSubHeading).toBeInTheDocument();
            expect(imageInput).toBeInTheDocument();
            expect(nameInput).toBeInTheDocument();
            expect(emailInput).toBeInTheDocument();
            expect(passwordInput).toBeInTheDocument();
            expect(confirmPasswordInput).toBeInTheDocument();
            expect(EditUserButton).toBeInTheDocument();
        });

        it("should render UserEdit with currentUser data", async () => {
            renderTheme(
                <BrowserRouter>
                    <UserContext.Provider
                        value={{
                            authenticated: true,
                            checkUser: mockCheckUser,
                        }}
                    >
                        <EditUser />
                    </UserContext.Provider>
                </BrowserRouter>
            );

            await waitFor(() => {
                expect(mockCheckUser).toHaveBeenCalledTimes(1);
            });

            const imageElm = screen.getByRole("img", { name: "john doe" });
            const nameInput = document.querySelector('input[name="name"]');
            const emailInput = document.querySelector('input[name="email"]');

            expect(imageElm).toHaveAttribute("src", "image.png");
            expect(nameInput).toHaveValue("john doe");
            expect(emailInput).toHaveValue("john@test.com");
        });

        it("should not render UserEdit component when authenticated is false", async () => {
            renderTheme(
                <BrowserRouter>
                    <UserContext.Provider
                        value={{
                            authenticated: false,
                            checkUser: mockCheckUser,
                        }}
                    >
                        <EditUser />
                    </UserContext.Provider>
                </BrowserRouter>
            );

            const formHeading = screen.queryByRole("heading", {
                name: "Editar Dados",
            });
            const formSubHeading = screen.queryByText(
                "Atualize suas informações pessoais"
            );

            const EditUserButton = screen.queryByRole("button", {
                name: "Atualizar",
            });

            expect(formHeading).not.toBeInTheDocument();
            expect(formSubHeading).not.toBeInTheDocument();
            expect(EditUserButton).not.toBeInTheDocument();
        });

        it("should match snapshot", async () => {
            const { container } = renderTheme(
                <BrowserRouter>
                    <UserContext.Provider
                        value={{
                            authenticated: true,
                            checkUser: mockCheckUser,
                        }}
                    >
                        <EditUser />
                    </UserContext.Provider>
                </BrowserRouter>
            );

            await waitFor(() => {
                expect(mockCheckUser).toHaveBeenCalledTimes(1);
            });

            expect(container.firstChild).toMatchSnapshot();
        });
    });
    describe("Functionality", () => {
        it("should call EditUser on button click", async () => {
            fetch.mockResponseOnce(JSON.stringify({ message: "Updated" }));

            renderTheme(
                <BrowserRouter>
                    <UserContext.Provider
                        value={{
                            authenticated: true,
                            checkUser: mockCheckUser,
                        }}
                    >
                        <EditUser />
                    </UserContext.Provider>
                </BrowserRouter>
            );

            await waitFor(() => {
                expect(mockCheckUser).toHaveBeenCalledTimes(1);
            });

            act(() => {
                fireEvent.click(
                    screen.getByRole("button", { name: "Atualizar" })
                );
            });

            await waitFor(() => {
                expect(fetch).toHaveBeenCalledTimes(1);
            });
        });
    });
});
