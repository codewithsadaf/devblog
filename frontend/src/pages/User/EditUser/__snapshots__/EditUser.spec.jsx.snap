// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserEdit /> Rendering should match snapshot 1`] = `
.c10 {
  padding: 0.5em 1.8em;
  font-size: 1em;
  border: none;
  font-weight: bold;
  cursor: pointer;
  background: #04a453;
  color: #fff;
  border: 2px solid #04a453;
  border-radius: 4px;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  text-transform: uppercase;
  width: 100%;
  padding: 1em 0;
}

.c10:hover {
  opacity: 0.8;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c0 {
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  width: 450px;
  margin-top: 3em;
}

.c1 > .c9 {
  margin-top: 1em;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-bottom: 1em;
}

.c7 {
  display: block;
  margin-bottom: 0.3em;
  font-weight: 700;
}

.c8 {
  padding: 7px 0;
  outline: none;
  border: none;
  border-bottom: 1px solid #ccc;
  background-color: transparent;
  font-size: 1em;
  -webkit-transition: all 500ms ease-in-out;
  transition: all 500ms ease-in-out;
}

.c8:focus {
  border-bottom: 1px solid #000;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c4 {
  font-size: 1em;
  line-height: 150%;
}

.c2 {
  margin-bottom: 1em;
}

.c3 {
  text-transform: none;
  font-weight: bold;
  font-size: 2.4em;
  line-height: 110%;
}

.c5 {
  width: 150px;
  height: 150px;
  object-fit: cover;
  display: block;
  margin: 1em auto;
  border-radius: 7px;
}

@media (max-width:768px) {
  .c1 {
    width: 100%;
    min-width: 0;
    max-width: 100%;
  }
}

<div
  class="c0"
>
  <form
    class="c1"
  >
    <div
      class="c2"
    >
      <h2
        class="c3"
      >
        Editar Dados
      </h2>
      <p
        class="c4"
      >
        Atualize suas informações pessoais
      </p>
    </div>
    <p
      class="c4"
    >
      <strong>
        Preview:
      </strong>
    </p>
    <div
      class=""
    >
      <img
        alt="john doe"
        class="c5"
        src="image.png"
      />
    </div>
    <div
      class="c6"
    >
      <label
        class="c7"
        for="image"
      >
        Foto de perfil:
      </label>
      <input
        class="c8"
        id="image"
        name="image"
        placeholder=""
        type="file"
        value=""
      />
    </div>
    <div
      class="c6"
    >
      <label
        class="c7"
        for="name"
      >
        Nome:
      </label>
      <input
        class="c8"
        id="name"
        name="name"
        placeholder="Nome do usuário"
        type="text"
        value="john doe"
      />
    </div>
    <div
      class="c6"
    >
      <label
        class="c7"
        for="email"
      >
        Email:
      </label>
      <input
        class="c8"
        id="email"
        name="email"
        placeholder="Email do usuário"
        type="email"
        value="john@test.com"
      />
    </div>
    <div
      class="c6"
    >
      <label
        class="c7"
        for="password"
      >
        Senha:
      </label>
      <input
        class="c8"
        id="password"
        name="password"
        placeholder="Insira sua senha"
        type="password"
        value=""
      />
    </div>
    <div
      class="c6"
    >
      <label
        class="c7"
        for="confirmpassword"
      >
        Confirme sua senha:
      </label>
      <input
        class="c8"
        id="confirmpassword"
        name="confirmpassword"
        placeholder="Confirmar senha"
        type="password"
        value=""
      />
    </div>
    <button
      class="c9 c10"
      type="submit"
    >
      Atualizar
    </button>
  </form>
</div>
`;
